### DISCUSSION TITLE
Duplicate IPs in Netbox

### DISCUSSION THREAD

**Discussion by Willdudeguy** [netbox-community/netbox#17250] [Help Wanted!]
I've recently started converting our IPAM into Netbox. I'm running into an issue with assigning interfaces  with IPs. As we are using VRRP to essentially have an active/passive VPN endpoint and the tunnel termination point is a Loopback, the IP for that loopback is of course assigned to both VPN endpoints. I want this to be reflected in Netbox but am not able to assign the same IP to both interfaces. Does anyone know if there is a method in which this can be achieved or should I do a feature request? Any help would be much appreciated. Thanks

**Comment by HB9HIL** [comment]
You want to create FHRP-Groups with the Virtual IP and assigning this FHRP Group to the related Interfaces.

**Comment by joaolucasmacedo** [comment]
> the IP for that loopback is of course assigned to both VPN endpoints. I want this to be reflected in Netbox but am not able to assign the same IP to both interfaces. 

Did you try to assign VRRP role to the IP address?
